### ISSUE TITLE
29237496: Accessibility Inspector fails to show object identity for accessibility elements

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#16335]
#### Description

Summary:
When using the Accessibility Inspector to inspect accessibility issues in iOS apps, the element section of the inspector fails to show the class or address of elements, as shown in the attached screenshot using the sample project provided (AccessibilityObjectIdentity).

Steps to Reproduce:
1. Launch an iOS app to inspect in the simulator
2. Launch the Accessibility Inspector and target the simulator
3. Inspect an element of interest

Expected Results:
The element's class and address are shown in order to ease debugging of accessibility issues.

Actual Results:
Note that the element does not show its class or address, while the accessibility label and traits are displayed accordingly.

Version:
macOS 10.12.1 (16B2657), Xcode 8.1 8B62, Simulator 10.0 (SimulatorApp-700.14), iOS 10.1 14B72

Notes:

Configuration:

Attachments:
'Screen Shot 2016-11-13 at 10.36.02.png' and 'AccessibilityObjectIdentity.zip' were successfully uploaded.

-
Product Version: 
Created: 2016-11-13T09:55:36.035000
Originated: 2016-11-13T00:00:00
Open Radar Link: http://www.openradar.me/29237496
